libs/ngx-mime/src/lib/viewer/viewer-footer/canvas-group-navigator/canvas-group-navigator.component.ts

```diff
@@ -48,7 +48,10 @@ export class CanvasGroupNavigatorComponent implements OnInit, OnDestroy {
       this.iiifManifestService.currentManifest.subscribe(
         (manifest: Manifest | null) => {
           if (manifest) {
-            this.invert = manifest.viewingDirection === ViewingDirection.LTR;
+            this.dir =
+              manifest.viewingDirection === ViewingDirection.LTR
+                ? 'ltr'
+                : 'rtl';
```

Comment:
Cannot value from enum of ViewingDirection be used here instead of hardcoding the value in the component?

Reply:
Yes. I have fixed it